src/pages/Batch/Batch.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import FadeIn from 'react-fade-in';
+import Styled from 'styled-components';
+import BestBatch from './BestBatch/BestBatch';
+import PeersBox from './PeersBox/PeersBox';
+import checkObjData from '../Util/checkObjData';
+
+export default function Batch() {
+  const [batchInfo, setBatchInfo] = useState({});
+  const { winner_batch_information, my_batch_information } = batchInfo;
+
+  useEffect(() => {
+    fetch('Data/BatchData.json')
+      .then(res => res.json())
+      .then(batchInfo => {
+        setBatchInfo(batchInfo.result[0]);
+      });
+  }, []);
+
+  return (
+    <FadeIn transitionDuration={1000}>
+      <Container>
+        {checkObjData(batchInfo) > 0 && (
```

Comment:
```suggestion
        {batchInfo && (
```
`checkObjData(batchInfo)` 리턴값이 true로 반환되던데, `> 0 ` 이 부분이 이해가 잘 안가는거같아요! 
이 부분 로직을 보면, 조건부 렌더링을 하시는 것으로 보입니다! 따라서 `state`값인 `batchInfo`가 있을때 렌더링 하는 로직은 어떠신가요??

Reply:
아 저게 기본 {}값이라 언제든지 true가 반환됩니당@.@ 그래서 불가피하게 함수 재사용을 해서나마 저렇게 번거롭게 구현하게 되었습닼ㅋㅋㅋ